### ISSUE TITLE
Is it necessary to install bdwgc from github?

### ISSUE THREAD

**Issue by zw963** [84codes/crystal-container-images#14]
https://github.com/84codes/crystal-container-images/blob/main/alpine/Dockerfile#L37-L43

Please check the [discuss about this](https://forum.crystal-lang.org/t/use-docker-cross-build-crystal-compiler-without-install-bdwgc-successful-but-why-shouldnt-bdwgc-be-a-required-dependency/6332/5) on the forum too.

Thanks.

**Comment by carlhoerberg** [comment]
Yes, we compile with other flags than gc-dev use. The performance with gc-dev is much worse, it isn't compiled with `--enable-large-config`.
